### ISSUE TITLE
Raspberry Pi 4 + Raspbian = Incompatible architecture arm detected

### ISSUE THREAD

**Issue by sebastian-burlacu** [nativefier/nativefier#1135]
Running latest Raspbian on Raspberry Pi 4b (4gb model)

After installing node + npm using debian instructions here https://github.com/nodesource/distributions/blob/master/README.md#debinstall

And installing nativefier without any errors using npm command, I can't build any images:

root@raspberrypi:~# nativefier web.whatsapp.com
Error during build. Run with --verbose for details. Error: Incompatible architecture arm detected
    at Object.inferArch (/usr/lib/node_modules/...

I'm writing this manually rather than copy pasting from the rpi so I won't include the whole error message but this is basically it.

**Comment by Itai-Nelken** [comment]
you have to add the `--arch=armv7l` (or `arm64` if you are using the 64bit beta), so for making a whatsapp web nativefier app you will have to run: `nativefier --arch="armv7l" web.whatsapp.com`.
also you will want to use the fix here: https://github.com/nativefier/nativefier/issues/1112#issuecomment-795424961 to make whatsapp web work correctly.

also don't run it as root.

**Comment by sebastian-burlacu** [comment]
thanks! the error message could be more descriptive... if it can detect that I'm using arm, why can't it suggest using this arch flag?

**Comment by ronjouch** [comment]
> the error message could be more descriptive... if it can detect that I'm using arm, why can't it suggest using this arch flag?

@sebastian-burlacu true! Thanks for the suggestion; renaming & re-opening the issue.
